mindmeld/components/entity_recognizer.py

```diff
@@ -79,6 +79,7 @@ def fit(self, queries=None, label_set=None, incremental_timestamp=None, **kwargs
         logger.info('Fitting entity recognizer: domain=%r, intent=%r', self.domain, self.intent)
 
         # create model with given params
+        kwargs['tokenizer'] = self._resource_loader.query_factory.tokenizer
```

Comment:
I think `Classifier._get_model_config()` overrides this parameter.

Reply:
resolved